### PR TITLE
1.30 release with resource fix

### PR DESCRIPTION
**Pull request by AndriiMoskvin**
## Changes

Supplying resource bundle instead of `resources` which sometimes work and sometimes not.

[JIRA Ticket](https://jira.ouroath.com/browse/OMSDK-2259)

@VerizonAdPlatforms/mobile-sdk-developers: Please review.

### CODE REVIEW

File: podspec/PlayerControls.podspec [hunk 1]
```diff
@@ -24,7 +24,7 @@ DESC
     :submodules => true 
   }   
   s.source_files     = 'PlayerControls/**/*.swift'
-  s.resources        = 'PlayerControls/resources/*'
+  s.resource_bundle  = { 'PlayerControls' => 'PlayerControls/resources/**/*.{xcassets,xib}' }
```
Comment (RomanTysiachnik) [inline podspec/PlayerControls.podspec]: 👍

File: PlayerControls/sources/BundleHelper.swift [hunk 2]
```diff
@@ -0,0 +1,21 @@
+//
+//  BundleHelper.swift
+//  PlayerControls
+//
+//  Created by amoskvin on 2/15/19.
```
Comment (RomanTysiachnik) [inline PlayerControls/sources/BundleHelper.swift]: Please, fix header ;)

File: PlayerControls/sources/DefaultControlsViewController.swift [hunk 3]
```diff
@@ -785,9 +786,9 @@ public final class DefaultControlsViewController: ContentControlsViewController
         
         airPlayView.props = AirPlayView.Props(
             icons: AirPlayView.Props.Icons(
-                normal: UIImage.init(named: "icon-airplay", in: Bundle(for: AirPlayView.self), compatibleWith: nil)!,
-                selected: UIImage.init(named: "icon-airplay-active", in: Bundle(for: AirPlayView.self), compatibleWith: nil)!,
-                highlighted: UIImage.init(named: "icon-airplay-active", in: Bundle(for: AirPlayView.self), compatibleWith: nil)!)
+                normal: UIImage.init(named: "icon-airplay", in: controlsBundle(for: AirPlayView.self), compatibleWith: nil)!,
```
Comment (VladyslavAnokhin) [inline PlayerControls/sources/DefaultControlsViewController.swift]: Can you use `createImage(named: )` from `Utils.swift`?

Comment (AndriiMoskvin) [reply PlayerControls/sources/DefaultControlsViewController.swift]: `view.createImage(named: "icon-airplay")` looks not so good...

File: PlayerControls/sources/BundleHelper.swift (hunk 2 above)
Comment (AndriiMoskvin) [reply PlayerControls/sources/BundleHelper.swift]: haha - ok

**Pull request merged**